src/SFML/Window/Sensor.cpp

```diff
@@ -0,0 +1,51 @@
+////////////////////////////////////////////////////////////
+//
+// SFML - Simple and Fast Multimedia Library
+// Copyright (C) 2007-2015 Laurent Gomila (laurent@sfml-dev.org)
+//
+// This software is provided 'as-is', without any express or implied warranty.
+// In no event will the authors be held liable for any damages arising from the use of this software.
+//
+// Permission is granted to anyone to use this software for any purpose,
+// including commercial applications, and to alter it and redistribute it freely,
+// subject to the following restrictions:
+//
+// 1. The origin of this software must not be misrepresented;
+//    you must not claim that you wrote the original software.
+//    If you use this software in a product, an acknowledgment
+//    in the product documentation would be appreciated but is not required.
+//
+// 2. Altered source versions must be plainly marked as such,
+//    and must not be misrepresented as being the original software.
+//
+// 3. This notice may not be removed or altered from any source distribution.
+//
+////////////////////////////////////////////////////////////
+
+////////////////////////////////////////////////////////////
+// Headers
+////////////////////////////////////////////////////////////
+#include <SFML/Window/Sensor.h>
+#include <SFML/Window/Sensor.hpp>
+#include <SFML/Internal.h>
+
+
+////////////////////////////////////////////////////////////
+sfBool sfSensor_isAvailable(sfSensorType sensor)
+{
+    return sf::Sensor::isAvailable(static_cast<sf::Sensor::Type>(sensor));
+}
+
+////////////////////////////////////////////////////////////
+void sfSensor_setEnabled(sfSensorType sensor, sfBool enabled)
+{
+    sf::Sensor::setEnabled(static_cast<sf::Sensor::Type>(sensor), enabled == sfTrue);
+}
+
+////////////////////////////////////////////////////////////
+sfVector3f sfSensor_getValue(sfSensorType sensor)
+{
+    sf::Vector3f value = sf::Sensor::getValue(static_cast<sf::Sensor::Type>(sensor));
+
+    return {value.x, value.y, value.z};
```

Comment:
No, this is brace initialization of an aggregate type (POD), this the same as when you initialize an array.